tools/f18/CMakeLists.txt

```diff
@@ -32,4 +35,30 @@ target_link_libraries(f18-parse-demo
   FortranParser
 )
 
+set_target_properties(f18 f18-parse-demo
+  PROPERTIES
+  RUNTIME_OUTPUT_DIRECTORY "${CMAKE_CURRENT_BINARY_DIR}/bin"
+)
+
+set(MODULES
+  "ieee_arithmetic"
+  "ieee_exceptions"
+  "ieee_features"
+  "iso_c_binding"
+  "iso_fortran_env"
+)
+
+# Create module files directly from the top-level module source directory
```

Comment:
Assumes that the module source file name matches the module file name.  Is this a safe assumption for now?

Reply:
It's a good assumption as long as we make the source file names match the module names. I don't think there is any reason not to do that.